roon-discord-publish.js

```diff
@@ -179,21 +158,42 @@ const roon = new RoonApi({
         _core = core;
         _transport = _core.services.RoonApiTransport;
 
+        let activeZone = null
         _transport.subscribe_zones((cmd, data) => {
-            const zoneOfInterest = _transport._zones[settings.zone.zone_id];
+            if (settings.zone_id) {
+                activeZone = _transport._zones[settings.zone_id];
+            }
+
+            // We have no zone, set it to the latest "playing"
+            // zone.
+            if (activeZone === null) {
+                for (const zoneID of Object.keys(_transport._zones)) {
+                    const zone = _transport._zones[zoneID]
+                    if (zone.state === 'playing') {
+                        activeZone = zone
```

Comment:
nit: (and for the rest of the lines): semicolons

Reply:
I DON'T BELIEVE IN SEMICOLONS